py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -145,6 +151,13 @@ def __parse_config(self, config):
             raise DataError(
                 "Missing argument 'use non-coadded spectra' required by DesiData"
             )
+        self.keep_single_exposures = config.getboolean(
+            "keep single exposures")
+        if self.keep_single_exposures is None:
+            raise DataError(
+                "Missing argument 'keep single exposures' required by DesiData"
+            )
+
```

Comment:
Yes I change this test. Do you mean that we should add a test mode for cross-exposure ? That is indeed a good idea. For now since the cross-exp is only preliminary, I suggest to postpone the test to the next PR

Reply:
I guess we should try to have multiple tests:
1) on the delta_extraction side tests for the different arguments allowed in the config now would be good so that future updates won't accidentally break this mode
2) a test using the outputs of delta_extraction for a well defined small dataset in the required settings as an input to the cross_pk routines and running those could be good to at least have a consistency check when putting more changes into the code